nikola/data/themes/bootstrap4/templates/gallery.tmpl

```diff
@@ -15,11 +15,37 @@
     </p>
     %endif
     %if folders:
+    % if galleries_use_thumbnail:
+    <div class="row">
+    % for folder, ftitle, fpost in folders:
+	  <div class="col-md-4 col-sm-6 col-xs-12" style="min-height: 425px;">
+	    <div class="thumbnail">
```

Comment:
I’m afraid that doesn’t work in bs4. The new `card` classes would have to be used.